src/components/AddSong/AddSong.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ChangeEvent, FormEvent, useState } from "react";
+import { useDispatch } from "react-redux";
+import { playlistsAsyncActions } from "../../store/slices/playlists/slice";
+import { Button, Container, FormContainer, HelperText, StyledInput, Title, Wrapper } from "./styles";
+
+const AddSong = () => {
+    const [url, setUrl] = useState<string>("");
+    const [helperText, setHelperText] = useState<string | null>(null);
+    const dispatch = useDispatch();
+
+    const saveSongUrl = (event: ChangeEvent<HTMLInputElement>) => {
+        let url = event.target.value;
+        if (url.length === 0) {
+            setHelperText("Cannot be empty!");
+            setUrl("");
+        } else {
+            setUrl(url);
+            setHelperText(null);
+        }
+    };
+
+    const submitSongUrl = (event: FormEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        console.log("will disp");
+        if (helperText ) {
+            return;
+        }
+
+        if (url.length === 0) {
```

Comment:
if the url lenght is 0, then the helpertext is not null right? so line is redundant

Reply:
When the component mounts, the length is still 0, but there is still no helper text. Only if the user writes sg, and then deletes the input.

Reply:
maybe set the helpertext to "cannot be null" right away?